app/src/main/AndroidManifest.xml

```diff
@@ -2,8 +2,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     package="com.stevesoltys.seedvault"
-    android:versionCode="7"
-    android:versionName="1.0.0">
+    android:versionCode="11000001"
```

Comment:
I thought we wanted to use API for versionCode, so `30000001`

Reply:
Good point, in case we need to target a possible `31` with a different version.

Reply:
Fixed.